elements/pfe-label/test/pfe-label.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { expect, html, oneEvent } from '@open-wc/testing';
+import { createFixture } from '@patternfly/pfe-tools/test/create-fixture.js';
+import { PfeLabel } from '@patternfly/pfe-label';
+
+const example = html`
+  <pfe-label></pfe-label>
+`;
+
+const exampleWithColorAttribute = html`
+  <pfe-label color="red"></pfe-label>
+`;
+
+const exampleWithOutColorAttribute = html`
+  <pfe-label color=""></pfe-label>
+`;
+
+const exampleWithOutlineAttribute = html`
+  <pfe-label outline></pfe-label>
+`;
+
+const exampleWithIconAttribute = html`
+  <pfe-label icon="rh-icon-virtual-storage-stack"></pfe-label>
+`;
+
+const exampleWithIconAttributeEmpty = html`
+  <pfe-label icon=""></pfe-label>
+`;
+
+/*
+* rgba2hex
+* https://stackoverflow.com/questions/1740700/how-to-get-hex-color-value-rather-than-rgb-value
+* @author Erick Petrucelli
+*/
+const rgba2hex = (rgba: string) => `#${rgba.match(/^rgba?\((\d+),\s*(\d+),\s*(\d+)(?:,\s*(\d+\.{0,1}\d*))?\)$/)?.slice(1).map((n: string, i: number) => (i === 3 ? Math.round(parseFloat(n) * 255) : parseFloat(n)).toString(16).padStart(2, '0').replace('NaN', '')).join('')}`;
```

Comment:
pfe-tools should provide this

https://github.com/patternfly/patternfly-elements/blob/main/tools/pfe-tools/test/hex-to-rgb.ts

Reply:
Awesome, missed that!

Reply:
the first, I think your function is valuable and should be in pfe-tools

Reply:
but if it's easier, the second is good too, your call